api/src/game/game.controller.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  Body,
+  Controller,
+  Get,
+  HttpException,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import { AppGateway } from '../app.gateway';
+import { GameState, Player } from '../types';
+import { GameService } from './game.service';
+import { INITIAL_BOARD } from '../constants';
+
+@Controller('game')
+export class GameController {
+  constructor(
+    private readonly gameService: GameService,
+    private gatewayService: AppGateway,
+  ) {}
+
+  @Post('start')
```

Comment:
posting on /games should be enough to create a game

Reply:
indeed, done